File: src/Actors/Administrador.java
package Actors;

public class Administrador extends Usuario{

    public Administrador(String nombre, String apellidos, String telefono, String identificacion, String correo, String contrasenia, String rol) {
        super(nombre, apellidos, telefono, identificacion, correo, contrasenia, "Administrador");
    }

}

File: Códigos/Actores/Administrador.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProyectoEstructuras.Códigos.Actores
{
    public class Administrador: Usuario
    {
        public Administrador(string nombre, string apellidos, string telefono, string identificacion, string numeroIdentificacion, string correo, string contrasenia)
            : base(nombre, apellidos, telefono, identificacion, numeroIdentificacion, correo, contrasenia, "Administrador")
        {
        }
    }
}
